generator.rb

```diff
@@ -0,0 +1,105 @@
+# method to test whether a letter is a vowel or consonant (found some help on https://www.ruby-forum.com/t/need-regex-to-check-for-consonants-and-put-number-after-each-consonant/224144)
+def is_vowel?(letter)
+  vowels = ["a", "e", "i", "o", "u"]
+  if vowels.include?(letter)
+    return true
```

Comment:
Good use of `include?` here. This list of letters might be a good place to use a constant.